### DISCUSSION TITLE
website templete

### DISCUSSION THREAD

**Discussion by shiinedev** [dugsiiyeinc/dhaqsoPay#3] [Ideas]
ka warama hadii an design kan websiteka  u isticmaalno  wixi fikrad ah waan so dhaweyna . qofkii design kale hayana ha soo shere gareyo asxaabta 🙌. sample design waye inte waad ka qiimenya karta aso website ah . https://online-peyment-design1.vercel.app
<img width="454" alt="online peyment" src="https://github.com/user-attachments/assets/db887803-263b-44e4-93ab-6af10fe7c003">

**Comment by sharafdin** [comment]
Ma xumo lkn waa uu yaryahay wax content-na maleh oo features ayuu ku egyahay sido kalena maleh pages kale. Mid kale aan raadino Insha'Allah!

**Reply by shiinedev** [reply]
Waa sax content badan kuma jiro template dheestiran qofki hayo hanala so wadaago

**Reply by sharafdin** [reply]
@dugsiiyeinc/dhaqsopay-contributors aan wada raadino Insha'Allah!
